src/components/PaperCheckout.tsx

```diff
@@ -67,6 +75,9 @@ export const PaperCheckout: React.FC<PaperCheckoutProps> = ({
   recipientWalletAddress,
   emailAddress,
   quantity,
+  eligibilityMethod,
+  mintMethod,
+  signatureArgs,
```

Comment:
Do we not need args for both eligibilityMethod and mintMethod?

Reply:
What do you mean by need args for both mint and eligibility?